pkg/kubelet/nodestatus/setters.go

```diff
@@ -49,6 +49,9 @@ const (
 	// MaxNamesPerImageInNodeStatus is max number of names
 	// per image stored in the node status.
 	MaxNamesPerImageInNodeStatus = 5
+
+	// ErrContainerRuntimeNotInitialized is reason for container runtime not initialized
+	ErrContainerRuntimeNotInitialized = "ContainerRuntimeNotInitializedError"
```

Comment:
Don't think we need a public constant here, just include it in the error message directly.